### ISSUE TITLE
[MacOS] Items in the lists do not hover on cursor hovering them

### ISSUE THREAD

**Issue by goodness-from-me** [jpochyla/psst#367]
**Describe the bug**
Items in the lists do not render as hovered upon mouse hovering over them. But when I do scroll the lists with mouse wheel the currently hovered item is properly marked as hovered.

**To Reproduce**
1. Having macOS 13.1 install the latest build of Psst.
2. Try to move mouse through track list or playlist list.
3. Observe list items are not rendered as "hovered" on mouse hovering over them.

**Expected behavior**
Items in the lists should properly hover over mouse being over them.

**Screenshots**

https://user-images.githubusercontent.com/42966950/213199712-54b194d2-397f-4411-92f5-c8b8b2fa0feb.mov

**Environment**
 - OS: macOS 13.1
 - Version: 2023-01-16 build

**Additional context**
I believe it's another issue that I can observe regarding rendering after #354 has been fixed.
I am not sure wether there was the same behaviour before #354 fix.

**Comment by goodness-from-me** [comment]
It's actually true for all the UI elements, including buttons. Edited bug description.

**Comment by jacksongoode** [comment]
Do you use a multimonitor setup @goodness-from-me ?

**Comment by goodness-from-me** [comment]
@jacksongoode No, I just use a MacBook with no external monitor.

**Comment by jacksongoode** [comment]
Does this still happen for you?

**Comment by jacksongoode** [comment]
Does this still happen for you?

**Comment by goodness-from-me** [comment]
Yes, it is still an issue on the latest version.

**Comment by Insprill** [comment]
Is this still an issue? There have since been some scrolling-related fixes in Druid (`linebender/druid#2149`) that I believe may have fixed this issue.

**Comment by goodness-from-me** [comment]
Yes, the rendering is still buggy and glitchy. But not so much as before. But every now and then on mouse hover or list scrolling items get disappeared and reappeared again in weird ways :( I can record new gifs if that helps.

**Comment by cloudybyte** [comment]
@goodness-from-me Are you by any chance using an Apple Silicon Mac?

The provided binary only supports x86 Macs and runs through Rosetta.
Building myself largely fixed the issue, but I have no idea why that could be. (There is still some flickering going on, but way **way** less than before)

**Comment by onpaws** [comment]
> (There is still some flickering going on, but way way less than before)

I'm on a Apple Silicon Mac and had the same experience building a binary myself. Some combo of Psst and a previous macOS release introduced some wild flickering, but as of #354 on macOS 13.5.2, today the flickering is mild and hardly a deal-breaker.

In my view, for a music app that's open all day, consuming <100MB RAM is definitely worth the tradeoff.
Thank you @jpochyla for your great work

**Comment by oleggtro** [comment]
I don't think there is any way to fix this flickering which would be worth the required effort

**Comment by literallyjustroy** [comment]
Still seeing this? I'm on M1 Mac on MacOS Sonoma 14.0 and I'm not experiencing any flickering/artifacting anymore.

**Comment by jacksongoode** [comment]
Really @literallyjustroy? I am still seeing this on M1 13.6 at least. I doubt anything would have changed since.... Have you tried moving the cursor around rapidly?

https://github.com/jpochyla/psst/assets/54308792/72dc9b3d-544a-4bca-946d-b9ee81d711e8

**Comment by literallyjustroy** [comment]
Yep, I used to see that behavior but not anymore.
![Screen Recording 2024-02-14 at 10 30 00 AM](https://github.com/jpochyla/psst/assets/56088145/24361302-ab47-4f99-834d-12ca63298462)
<img width="296" alt="image" src="https://github.com/jpochyla/psst/assets/56088145/b18a1c20-9d97-4f70-bf50-621512858ea1">

**Comment by onpaws** [comment]
Anecdata: In Oct 22 on Intel I [noticed](https://github.com/jpochyla/psst/issues/354) "lots" of flickering. Later in 2023 on an Apple Silicon machine, the flickering dwindled - still there but much less. Today in 2024 I haven't noticed it in at least a few months, and despite my efforts wiggling the mouse right now can no longer repro flickering at all. No clue why 🤷 

M2 Max / Sonoma 14.3.1

**Comment by jacksongoode** [comment]
Can anyone else can reproduce this on macOS 13.x?
